index.src.html

```diff
@@ -70,6 +70,9 @@
       text: Navigator
 spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
+    text: same-origin; url: concept-origin-effective-domain
```

Comment:
This should link to `same-origin`, not `concept-origin-effective-domain`. I'm surprised it's not showing up automatically... The definition should be exported based on the HTML spec's markup...